README.rst

```diff
@@ -694,6 +694,10 @@ This means:
 -  It is not possible to add methods for more than one selector to a right inlet.
    Particularly, it is not possible to add a universal method for arbitrary selectors to a right inlet.
 
+Note however than if ``inlet_new`` is invoked with the ``list`` symbol as selector, then 
+as a convenience feature, both ``float`` and ``symbol`` messages will [also be supported](https://sourceforge.net/p/pure-data/patches/164/),
+but never arbitrary messages. 
+
```

Comment:
hmm.
i'd rather not link to 17 year old PR that was accepted 10 years ago on a platform we no longer use.
(in general, the documentation does not link to commits,... that only show when/how a given functionality was implemented).